README.md

```diff
@@ -57,47 +57,17 @@ Clone the repository and change working directory (if not already) then build th
 $ git clone https://github.com/oduwsdl/tmvis.git
 $ cd tmvis
 $ docker image build -t timemapvis .
+$ docker run --shm-size=1G -it --rm -v "$PWD":/app -v /app/node_modules -p 3000:3000 --user=$(id -u):$(id -g) timemapvis node tmvis.js
```

Comment:
```suggestion
$ docker run --shm-size=1G -it --rm -p 3000:3000 timemapvis
```

To run the application in a simple mode, you do not need any of that extra stuff. You might document the location of the cache/screenshots directory so people can bind mount that volume for persistence, but you do not want them to mount the `$PWD` in `/app` which is the use case of development mode.

Reply:
This would make the code block match the non development mode block defined earlier in the file. Do you think it would be better to get rid of the non development block to avoid confusion and rename the heading of the development mode to say "Running as a Docker Container"?

Reply:
There should be just one section for Docker in which you start from a simple use case, then talk about necessary volumes for persistent storage, and finally discuss what is needed if developers want to make their local changes reflected without rebuilding the image each time (unless a dependency changes). The whole naive and experimental annotation is unnecessary and confusing.

Reply:
Gotcha, I'll get rid of that and make appropriate changes. Thanks!